engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java

```diff
@@ -61,6 +55,10 @@ public static DataflowError withoutTrace(Object payload, Node location) {
     }
   }
 
+  public static DataflowError withoutTrace(Object payload, Node location) {
```

Comment:
This overload is here for the calls to `withoutTrace` which originate in Java and cannot pass a `State` value. This will be removed in part 2 which implements the same functionality for such calls.